tests/scripts/thread-cert/Cert_5_1_01_RouterAttach.py

```diff
@@ -168,6 +179,317 @@ def test(self):
         for addr in self.nodes[LEADER].get_addrs():
             self.assertTrue(self.nodes[ROUTER].ping(addr))
```

Comment:
The test plan actually requires pining DUT's link local address. 
```
Verify connectivity by
sending an ICMPv6 Echo
Request to the DUT link
local address. 
```

So, we might only need to ping the link local address rather than pinging each address?
Also, I think we should also ping ROUTER from LEADER because both ROUTER and LEADER can be the DUT device.

Reply:
added another ping for both DUT roles